nixos/modules/virtualisation/containers.nix

```diff
@@ -518,6 +518,7 @@ in
         KillSignal = "WINCH";
 
         DevicePolicy = "closed";
+        DeviceAllow = "/dev/net/tun rw";
```

Comment:
Hmm, problem is that I can't have multiple `DeviceAllow` lines, since nix doesn't allow multiple attributes with the same name.